docker-compose.yml

```diff
@@ -1,6 +1,10 @@
 version: '3'
 services:
-  postgres:
+  lmspostgres:
```

Comment:
Name it something different than the h postgres container or the names will conflict and docker dns will randomly chose one.